cpp/benchmarks/string/url_decode.cu

```diff
@@ -67,15 +67,15 @@ auto generate_column(cudf::size_type num_rows, cudf::size_type chars_per_row, do
   auto col_1a     = cudf::test::strings_column_wrapper(strings.begin(), strings.end());
   auto table_a    = cudf::repeat(cudf::table_view{{col_1a}}, num_rows);
   auto result_col = std::move(table_a->release()[0]);  // string column with num_rows  aaa...
-  auto chars_col  = result_col->child(cudf::strings_column_view::chars_column_index).mutable_view();
+  auto chars_col  = result_col->mutable_view();
   auto offset_col = result_col->child(cudf::strings_column_view::offsets_column_index).view();
 
   auto engine = thrust::default_random_engine{};
   thrust::for_each_n(thrust::device,
                      thrust::make_zip_iterator(offset_col.begin<cudf::size_type>(),
                                                offset_col.begin<cudf::size_type>() + 1),
                      num_rows,
-                     url_string_generator{chars_col.begin<char>(), esc_seq_chance, engine});
+                     url_string_generator{chars_col.data<char>(), esc_seq_chance, engine});
```

Comment:
This looks odd to me. I had not considered how accessing the chars data directly from the strings column would appear. Also, the `data()` function adds the `offset()` value and so this would not generally be correct (although it works here since we know the column is not sliced). I think I might prefer something like this instead:
```
   static_cast<char*>(chars_col.head());
```
Let me know what you think.

Reply:
Sounds good. Replaced all occurances with this suggestion.